pkg/statsd/handler_cloud_test.go

```diff
@@ -72,20 +73,31 @@ func TestTransientInstanceFailure(t *testing.T) {
 	defer wg.Wait()
 	ctx, cancelFunc := context.WithCancel(context.Background())
 	defer cancelFunc()
+	clck := clock.NewMock(time.Unix(0, 0))
```

Comment:
isn't this the same as `time.Time{}`?

Reply:
I guess. Does it matter?

Reply:
So quickly checking, `time.Unix(...)` will return time since the start of EPOCH, `time.Time{}` returns zero time.

> Does it Matter?
Not really, more trying understand why you're doing this.

Reply:
The actual time doesn't matter, it's all relative based time. `time.Time{}` would work as well, since there's no checks for zero, and the virtual clock still advances.

I did it because my IDE was like "hey, this call needs a time", and I'm not very creative :p